api/v1/version.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package v1;
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/mt-sre/addon-service/api/v1";
+
+message Version {
+  string version = 1;
+  string branch = 2;
+  string commit = 3;
+  string goVersion = 4;
+  string platform = 5;
+  google.protobuf.Timestamp buildTime = 6;
```

Comment:
Tiny nit-pick: do we want to instead stick to snake-casing message field names? i.e; `s/goVersion/go_version` and `s/buildTime/build_time`?

As per the Google Protobuf style guide: https://developers.google.com/protocol-buffers/docs/style

Reply:
Should we use a proto-linter like: https://github.com/yoheimuta/protolint to enforce conventions?